build.sbt

```diff
@@ -237,6 +236,9 @@ def multiJvm(project: Project): Project = {
         Seq(
           parallelExecution in Test := false,
           parallelExecution in MultiJvm := false,
+          // -o D(report the duration of the tests) F(show full stack traces)
+          // -u select the JUnit XML reporter
+          scalatestOptions in MultiJvm := Seq("-oDF", "-u", (target.value / "test-reports").getAbsolutePath),
```

Comment:
@dwijnand as we discussed, `multi-jvm` tests were not generating reports. This is probably something we want to fix in `MultiJvmPlugin` later but for now, this fixes the problem on our end.